src/services/codefixes/importFixes.ts

```diff
@@ -0,0 +1,474 @@
+﻿/* @internal */
+namespace ts.codefix {
+
+    type ImportCodeActionKind = "CodeChange" | "InsertingIntoExistingImport" | "NewImport";

```

Comment:
what does these string do? are they constants as part of the API? what other types do we have?